model/supervised_finetuning/models/gptj.py

```diff
@@ -0,0 +1,187 @@
+# Taken from https://github.com/sleekmike/Finetune_GPT-J_6B_8-bit/blob/master/gpt-j-6b-8-bit.py
+
+import torch
```

Comment:
Maybe instead use bitsandbytes for a generic solution for 8-bit quantization? Are there any downsides to that?

Reply:
Not really! feel free to edit

Reply:
Also I'm pretty sure I've previously seen a generic library that implements a whole pile of different adapter types, could be convenient.

Reply:
@sanagno Opened PR [here](https://github.com/LAION-AI/Open-Assistant/pull/582)